tools/boilerplate.py

```diff
@@ -141,7 +141,8 @@ def generate_function(name, called_fullname, template, **kwargs):
               and param.default is Parameter.empty else
            # Only pass the data kwarg if it is actually set, to avoid forcing
            # third-party subclasses to support it.
-           '**({{"data": data}} if data is not None else {{}})'
+           '**({{"data": data}} if data is not None else {{}})'.replace(
+               ' ', '\0')  # Avoid linebreaks in the middle of the expression.
```

Comment:
Please comment explicitly that \0 is used as a placeholder and will be replaced later on.

Reply:
done